api/src/service/email-service.ts

```diff
@@ -0,0 +1,38 @@
+import nodemailer, { SendMailOptions } from 'nodemailer';
+import {EMAIL_HOST, EMAIL_PORT, EMAIL_USER, EMAIL_PASS} from '../config/keys';
+
+
+export const sendEmail = async (to: string, subject: string, msg: string, icsFileContent?: string) => {
+    try {
+        const transporter = nodemailer.createTransport({
+            host: EMAIL_HOST,
+            port: EMAIL_PORT,
+            secure: true,
+            auth: {
+                user: EMAIL_USER, 
+                pass: EMAIL_PASS 
+            }
+        });
+        
+        const emailOptions = <SendMailOptions> {
+            from: `"Vita" <${EMAIL_USER}>`,
+            to,
+            subject,
+            text: msg
+        }
+        
+        if (icsFileContent)
+            emailOptions.attachments = [
+                {
+                    filename: 'Event.ics',
+                    content: icsFileContent
+                }
+            ]
```

Comment:
Can we change these lines of code to these please ?
`emailOptions.icalEvent = {
    content: icsFileContent,
    method: 'request'
}`

For some reason when I tested your email service and ics service, The mail was working perfectly but ics file attachment didn't showed this preview for the ics file:
![Screenshot 2022-04-05 164628](https://user-images.githubusercontent.com/84276404/161742476-bf03fa6e-422f-4445-8f7b-99028d394dbc.png)

But when I changed the above mentioned code, the preview showed up.